script/bigscreenplayer.js

```diff
@@ -270,13 +286,19 @@ define('bigscreenplayer/bigscreenplayer',
           pauseTrigger = opts && opts.userPause === false ? PauseTriggers.APP : PauseTriggers.USER;
           playerComponent.pause(opts);
         },
-        setSubtitlesEnabled: function (value) {
-          playerComponent.setSubtitlesEnabled(value);
-          callSubtitlesCallbacks(value);
+        resize: function (top, left, width, height, zIndex) {
+          subtitlesHidden = isSubtitlesEnabled();
+          setSubtitlesEnabled(false);
+          resizer.resize(playbackElement, top, left, width, height, zIndex);
         },
-        isSubtitlesEnabled: function () {
-          return playerComponent ? playerComponent.isSubtitlesEnabled() : false;
+        clearResize: function () {
+          if (subtitlesHidden) {
```

Comment:
Can we not just use isSubtitlesEnabled() to turn them back on instead of having more state with subtitlesHidden?

Reply:
No, because the `resize` function makes these false, so if we just used `isSubtitlesEnabled` when clearing the resize, this would always be false.